makefile

```diff
@@ -439,6 +440,12 @@ tests-e2e-setup: setup-cluster
 		-e ARMADA_APICONNECTION_ARMADAURL="server:50051" \
 		-e ARMADA_APICONNECTION_FORCENOTLS=true \
 		armada-executor --config /e2e/setup/insecure-executor-config.yaml
+	docker run -d --name lookout-ingester-migrate  --network=kind -v ${PWD}/e2e:/e2e \
+		armada-lookout-ingester --config /e2e/setup/lookout-ingester-config.yaml --migrateDatabase
+	sleep 3
```

Comment:
Is there anything we can check to see if this job worked without sleeping?

Reply:
assuming this runs synchronously I'm not sure why we need to sleep here? As soon as the process exists we know that the db is in a good state. Is it to make sure the logs are available.